Bucketeer/Sources/Internal/Model/MetricsEventType.swift

```diff
@@ -1,10 +1,9 @@
 import Foundation
 
 enum MetricsEventType: Int, Codable, Hashable {
+    case unknownError = 0
     case responseLatency = 1
     case responseSize = 2
-//    case timeoutErrorCount = 3 // deprecated
-//    case internalErrorCount = 4 // deprecated
     case timeoutError = 5
```

Comment:
Is it possible to re-order the values? From 3, 4, 5...

Reply:
Reordered and Use default enum value. (Swift's Int enum starts at `0` by default. )
https://github.com/ca-dp/bucketeer-ios-sdk/pull/92/commits/75c0f0869f63aa74a2cabc2caf5aa76c6785c97b